### ISSUE TITLE
Esp32 compatibility

### ISSUE THREAD

**Issue by ShiveStar** [T-vK/alexa-jammer#3]
Hi,

In your description of the alexa wifi jammer you say you need either a esp32 or esp8266, however, looking over the code it seems only the later is supported. Is the esp32 going to be supported in a future revision or is that a deprecated feature? Thanks for the cool project!

**Comment by T-vK** [comment]
You're right, I think it would be an easy fix to make it work with an ESP32 as well. I'll see when I can find some time to work on that. Thanks for reporting.

**Comment by anmayraj** [comment]
i hope you will add the feature for esp32 soon
we are waiting

**Comment by Chrispyfur1904** [comment]
Hi, I cannot get this to work with an ESP32. It throws up a number of errors. I have ordered a eps8226. However... Can you help?
